cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -2764,3 +2765,107 @@ def test_root_volume_encryption_consistency_validator(
         assert_failure_level(actual_failures, FailureLevel.WARNING)
     else:
         assert_that(actual_failures).is_empty()
+
+
+@pytest.mark.parametrize(
+    "queues, num_cards, public_ip_subnets, expected_error_messages",
+    [
+        pytest.param(
+            [
+                SlurmQueue(
```

Comment:
You can simplify test parameters having only the assign_public_ip parameter and build the SlurmQueue in the test code, since all the other fields are the same for all the tests (except the latest one, but you might use two subnet in the latest test too).